presto-mongodb/src/main/java/com/facebook/presto/mongodb/MongoPageSource.java

```diff
@@ -314,4 +317,15 @@ public void close()
     {
         cursor.close();
     }
+
+    @Override
+    public void deleteRows(Block rowIds)
```

Comment:
Does this get called during delete?

Reply:
Yes, this gets called, but it doesn't delete any rows from this method. Delete rows will happen in deleteDocuments method. This method is overridden to overcome the error `java.lang.UnsupportedOperationException: This connector does not support row-level delete`

Reply:
Please add a comment, this is very unintuitive.